docs/book/src/introduction.md

```diff
@@ -0,0 +1 @@
+{{#include ../../../README.md}}
```

Comment:
While intro page is displayed correctly using this, clicking the reference links to getting-started.md or development.md from the root README file is not redirecting me to the correct locations (`Document not found (404)`). 

FYI, I tried moving the root README file contents here and fixed the paths to above mentioned files, all looks fine after that